packages/flame/lib/src/components/component.dart

```diff
@@ -732,3 +779,24 @@ class _LifecycleManager {
     }
   }
 }
+
+/// A simple tuple of a component and a point. This is a helper class for
+/// [Component.componentsAtPoint] method.
+@immutable
+class ComponentPoint {
```

Comment:
I think this should live in its own file

Reply:
moved